src/containers/ArticlePage/LearningResourcePage/components/LearningResourcePanels.tsx

```diff
@@ -50,21 +50,30 @@ const LearningResourcePanels = ({
   const { errors } = useFormikContext<LearningResourceFormType>();
   const defaultOpen = useMemo(() => ["learning-resource-content"], []);
 
-  const contentTitleFields = useMemo<(keyof IArticle)[]>(() => ["title", "introduction", "content"], []);
-  const copyrightFields = useMemo<(keyof IArticle)[]>(() => ["copyright"], []);
+  const lastPublishedVersion = articleHistory?.find((a) => a.status.current === PUBLISHED);
+
+  const contentTitleCompareData = useMemo(
+    () => [
+      { current: article?.title?.title, published: lastPublishedVersion?.title?.title, isHtml: true },
+      { current: article?.content?.content, published: lastPublishedVersion?.content?.content, isHtml: true },
+      {
+        current: article?.introduction?.introduction,
+        published: lastPublishedVersion?.introduction?.introduction,
+        isHtml: true,
+      },
+    ],
+    [article, lastPublishedVersion],
+  );
```

Comment:
Ser ut som at vi gjør dette ganske mange steder. Hadde det gått an å gjøre det mer gjenbrukbart? Synes forsåvidt den gamle løsningen var ganske grei. Kunne oppdatert den til å støtte nested keys.

Reply:
Hovedproblemet med den gamle løsningen var å få det til å fungere for html når kommentarer må filtreres ut 😅 Men det er mulig jeg har overkomplisert det her, kunne prøvd og støttet nestede keys, også lage en sjekk på om innholdet er html eller ikke før vi filtrerer vekk kommentarer ..